modules/migration-mtc-release-notes-1-7-19.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes-1-7.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-19_{context}"]
+= {mtc-full} 1.7.19 release notes
+
+{mtc-first} 1.7.19 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers. No code was changed in the product itself compared to that of {mtc-short} 1.7.18.
+
+[id="resolved-issues-1-7-19_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.MTC installation fails due to an Ansible Runner KeyError
```

Comment:
The current phrasing might be misleading. This install failure will not happen with MTC ver 1.7.19.
What we need to convey here is:

In 1.7.19 the `python3-requests` package was not upgraded above `python3-requests-2.20.0-3.el8_8.noarch` since it is causing MTC install failure (provide the below install failure details). 
This action does mean exposure to the following vulnerability, link:https://access.redhat.com/errata/RHSA-2023:4520[RHSA-2023:4520].